draft-ietf-quic-transport.md

```diff
@@ -5571,6 +5571,12 @@ Retire Prior To field MUST be less than or equal to the Sequence Number field.
 Receiving a value greater than the Sequence Number MUST be treated as a
 connection error of type FRAME_ENCODING_ERROR.
 
+Endpoints that receive a NEW_CONNECTION_ID frame SHOULD NOT retire connection
```

Comment:
I honestly find this (and that it is normative language) more confusing than not: it's incredibly specific about the requirement, and that requirement is the exact inverse requirement of Retire Prior To -- I get that that's the point, but the phrasing leads me down a confusing path. 
I went to suggest an alternative for wording (and a new location away from the frame mechanics and as the second to last paragraph of `{{retiring-cids}}`), but I'm struggling to find anything here that doesn't seem to cause additional confusion. 

I'm not quite seeing how any reading of the Retire _PRIOR_ To text leads one to try and retire all of the CIDs that aren't _PRIOR_ to the value received. From the description of the issue you ran into, the problem was less around RPT at all and more than you tried to throw away all other CIDs when you got a NCID frame, which might be something worth mentioning but is not this text.

Text more to that effect would probably go best around the active_cid_limit discussion where we indicate that you should probably keep as many as you said you would be willing for the peer to send you.

Reply:
I am happy to jettison this paragraph to get the other bit through. The point is to say "here is a stupid thing to do and don't do it". If there's some other wording, or some other place to put it that's better, I'm fine with that too.